src/shared/src/components/Select.tsx

```diff
@@ -0,0 +1,71 @@
+import { makeStyles } from '@material-ui/core/styles';
+import cn from 'classnames';
+import * as React from 'react';
+import { AltinnAppTheme } from '../theme';
+import '../../../altinn-app-frontend/src/styles/shared.css';
+
+const useStyles = makeStyles({
+  select: {
+    fontSize: '1.6rem',
+    '&:focus': {
+      outline: `2px solid ${AltinnAppTheme.altinnPalette.primary.blueDark}`,
+    },
+  },
+});
+
+const optionStyle = {
+  display: 'none',
+};
+interface ISelectProps {
+  id: string;
+  onChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  onBlur?: (event: React.FocusEvent<HTMLSelectElement>) => void;
+  options: { value: string; label: string }[];
+  value: string;
+  disabled?: boolean;
+  error?: boolean;
+  fullWidth?: boolean;
+  className?: string;
+}
+
+const Select = ({
+  id,
+  onChange,
+  onBlur,
+  options,
+  value,
+  disabled = false,
+  error = false,
+  fullWidth = false,
+  className,
+}: ISelectProps) => {
+  const classes = useStyles();
+  return (
+      <select
+        id={id}
+        value={value}
+        disabled={disabled}
+        className={cn(
+          classes.select,
+          'custom-select a-custom-select',
+          {
+            'validation-error': error,
+            'disabled !important': disabled,
```

Comment:
does this work?

Reply:
This is also something I copyd the and does actually work. The Select looks like this: 
![image](https://user-images.githubusercontent.com/43168787/169825192-e8fba007-be7d-4b78-bcfe-d50e036227f4.png)